package.json

```diff
@@ -5,16 +5,17 @@
     "start": "start-storybook -p 9009",
     "eslint": "eslint ./src",
     "eslint:fix": "eslint --fix ./src",
-    "build": "nwb build-react-component",
-    "build:dist": "nwb build-react-app",
+    "build": "npm run build:lib && npm run build:es",
+    "build:lib": "rimraf lib && babel src -d lib",
+    "build:es": "rimraf es && babel --config-file ./config/babel/es.json src -d es",
+    "build:dist": "webpack -p es/index.js dist/index.js",
     "clean": "rimraf lib && rimraf es",
     "flow": "flow",
     "bundlesize": "bundlesize",
     "test": "CI=true npm run test:unit && npm run eslint && npm run flow",
-    "test:unit": "node scripts/test.js --env=jsdom",
-    "test:debug": "node  --inspect-brk scripts/test.js --runInBand --env=jsdom",
+    "test:unit": "jest --env=jsdom",
+    "test:debug": "node  --inspect-brk ./node_modules/.bin/jest --env=jsdom",
     "test:snapshot": "npm run test:unit -- -u",
-    "coverage": "npm run test:unit -- --coverage",
```

Comment:
did you intend to remove this?

without it, is it still possible to run a local test with a coverage report?

Reply:
yes, because our jest config in package.json:

```
    "coverageDirectory": "./coverage/",
    "collectCoverage": true,
```

means that test coverage is always obtained by default